test/integration/integration_test.go

```diff
@@ -279,31 +288,57 @@ func checkConfigKey(configPath string, condition string, keyPath string) error {
 
 func compareExpectedWithActualContains(expected string, actual string) error {
 	if !strings.Contains(actual, expected) {
-		return fmt.Errorf("Output did not match. Expected: %s, Actual: %s", expected, actual)
+		return fmt.Errorf("Output did not match. Expected: '%s', Actual: '%s'", expected, actual)
```

Comment:
@hferentschik Glad you like '%s'. I looked onto errors.New vs fmt.Errorf topic, if I understand it correctly errors.New does not support formatting, but if you do not need it then it saves one extra call of fmt.Sprintf(), fmt.Errorf supports formatting, which is needed here. Went through integration_test.go, util.go and minishift.go in test folder and found one fmt.Errorf() call, which do not use formatting and can be turned into more effective errors.New(). Going to fix it.